File: system-test/dotnet/SystemTests/Legacy/Mod07/SmokeTests/External/TaxSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod07.Base;
using Shouldly;
using Xunit;

namespace SystemTests.Legacy.Mod07.SmokeTests.External;

public class TaxSmokeTest : BaseSystemDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToTax()
    {
        (await _app.Tax().GoToTax()
            .Execute())
            .ShouldSucceed();
    }
}

File: system-test/java/src/test/java/com/optivem/shop/systemtest/legacy/mod07/smoke/external/TaxSmokeTest.java
package com.optivem.shop.systemtest.legacy.mod07.smoke.external;

import com.optivem.shop.systemtest.legacy.mod07.base.BaseUseCaseDslTest;
import org.junit.jupiter.api.Test;

class TaxSmokeTest extends BaseUseCaseDslTest {
    @Test
    void shouldBeAbleToGoToTax() {
        app.tax().goToTax()
                .execute()
                .shouldSucceed();
    }
}
